test/test_torch.py

```diff
@@ -7290,6 +7290,55 @@ def run_test(p):
         with self.assertRaisesRegex(RuntimeError, "chain_matmul: Expected one or more matrices"):
             torch.chain_matmul()
 
+    # no CUDA LU yet, used for torch.det())
+    @onlyCPU
+    @skipCPUIfNoLapack
```

Comment:
@mruberry should this test go in `test_unary_ops` or `test_linalg`?

Reply:
test_linalg

Reply:
alright then @nikitaved we should just update this test: https://github.com/pytorch/pytorch/blob/8e8fb8542e806b4c966c6c04bc847f915abac61f/test/test_linalg.py#L162

Reply:
Hmm, when I try to move things into `test_linalg.py`, some checks fail, unlike for `test_torch.py`.

Reply:
All fine, the tests are for quite large tensors, so we run into overflow issues.